Makefile

```diff
@@ -32,6 +32,7 @@ upgrade: upgrade-piptools piptools ## upgrade requirement pins.
 	sed 's/pyjwt\[crypto\]<2.0.0//g' requirements/common_constraints.txt > requirements/common_constraints.tmp
 	mv requirements/common_constraints.tmp requirements/common_constraints.txt
 	pip-compile --allow-unsafe --rebuild -o requirements/pip.txt requirements/pip.in
+	pip install -qr requirements/pip.txt
```

Comment:
We should move all commands inside `make upgrade` target to keep the changes consistent among all repos.